src/nodes/display/PassNode.js

```diff
@@ -130,7 +131,18 @@ class PassNode extends TempNode {
 
 	}
 
-	setup() {
+	setup( { renderer } ) {
+
+		this.renderTarget.samples = this.options.samples === undefined ? renderer.samples : this.options.samples;
+
+		this.renderTarget.depthTexture.isMultisampleRenderTargetTexture = this.renderTarget.samples > 1;
+
+		for ( let i = 0; i < this.renderTarget.textures.length; i ++ ) {
+
+			const type = this.renderTarget.textures[ i ].type;
+			this.renderTarget.textures[ i ].isMultisampleRenderTargetTexture = this.renderTarget.samples > 1 && type !== HalfFloatType && type !== FloatType;
```

Comment:
What is the purpose of `type !== HalfFloatType && type !== FloatType`? 

Does that mean multisampled (half) float textures are not allowed?